File: Programming Fundamentals with Python/Functions/grades.py

def receive_grade(input_grade) -> float:
    if 2.00 <= input_grade <= 2.99:
        return "Fail"
    elif 3.00 <= input_grade <= 3.49:
        return "Poor"
    elif 3.50 <= input_grade <= 4.49:
        return "Good"
    elif 4.50 <= input_grade <= 5.49:
        return "Very Good"
    elif 5.50 <= input_grade <= 6.00:
        return "Excellent"


input_grade = float(input())

result = receive_grade(input_grade)
print(result)

File: Programming Fundamentals with Java/Methods/Grades.java
import java.util.Scanner;

public class Grades 
{
    public static String printGrade(double grade)
    {
        if (2 <= grade && grade <= 2.99)
        {
            return "Fail";
        }
        else if (3 <= grade && grade <= 3.49)
        {
            return "Poor";
        }
        else if (3.5 <= grade && grade <= 4.49)
        {
            return "Good";
        }
        else if (4.5 <= grade && grade <= 5.49)
        {
            return "Very good";
        }
        else if (5.5 <= grade && grade <= 6)
        {
            return "Excellent";
        }
        return null;
    }

    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        double grade = Double.parseDouble(input.nextLine());
        System.out.println(printGrade(grade));
    }
}
